java/patterns/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,12 @@
+# Set to debug or trace if log4j initialization is failing
```

Comment:
Please use the new properties from https://github.com/restatedev/sdk-java#logging

Reply:
Will update it.